src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -116,34 +144,67 @@ void EnHoll_SwapRooms(PlayState* play) {
     play->roomCtx.unk_30 ^= 1;
 }
 
-// Horizontal Planes
-void func_80A58DD4(EnHoll* this, PlayState* play) {
+/**
+ * When traversing a holl of this kind, it attempts to:
+ *   Load Current Room (fails as it is already loaded)
+ *   Load Next Room
+ *   Load Previous Room
+ *   Load Next Room
+ *
+ *  @bug The striped nature of holls can cause some actors to unload due to
+ *      conflicting Object Lists between the two rooms
+ *
+ *  @bug If you can get around to the other side of the holl without triggering it,
+ *      you can load the room on the other side multiple times
+ */
+void EnHoll_HorizontalVisibleNarrow(EnHoll* this, PlayState* play) {
+    /**
+     * These are all absolute distances in the relative z direction. That is, moving
+     *   towards or away from the "face" of the holl regardless of orientation.
+     * Moving within these distances of the holl have the following effects:
+     * [0] : Load the room on this side of the holl if not already loaded
+     * [1] : Load the room on the other side of the holl
+     * [2] : Start of fade region, where the plane is fully opaque
+     * [3] : End of fade region region, where the plane is fully transparent
+     *
+     * Within the fade region, the plane goes:
+     *   opaque -> transparent if approaching,
+     *   transparent -> opaque if receding
+     */
+    static f32 sTriggerDists[2][4] = {
+        { 200.0f, 150.0f, 100.0f, 50.0f }, // default
+        { 100.0f, 75.0f, 50.0f, 25.0f },   // SCENE_JYASINZOU
+    };
```

Comment:
does this need to be in-static to match? if not id be ok with moving out just for the sake of not cluttering the function itself with the large explanation of the data

Reply:
It can be outside of the function, it is in master

I thought it was better inside it as only the `EnHoll_HorizontalVisibleNarrow`/`ENHOLL_H_VISIBLE_NARROW` type of holl makes use of this data, and it would be better to not think it's used by more types

I can put it outside and name it `sHorizontalVisibleNarrowTriggerDists` to address that too?

Also if I do move it outside, should it be near the top of the file, or is just above the function fine as well

Reply:
I would personally do it just above the function. but yeah this is a huge preference thing that we have no official guidelines for

I just personally dont like it when data+comments is taking up huge chunks of a function, i want the function to just get to the code asap